web-app/src/api/keyManager.js

```diff
@@ -45,7 +48,8 @@ const loadOrCreate = () => {
     let credentials = isSaved();
     if (!credentials) {
         credentials = generate();
-        localStorage.setItem('credentials', JSON.stringify(credentials))
+        saveCredentials(credentials)
```

Comment:
I think we are breaking the verified:false logic here by saving credentials to local storage anyway.

Reply:
We only use that method in account generation, not in recovery or checking. It does not collide with the other functionality.

Reply:
I got confused with the collapsing interface of github.